Description: Relax build dependencies
 This patch relax the build dependencies such that the package can be easier built.
Author: Johann Felix Soden <johfel@debian.org>
Forwarded: not-needed
Last-Update: 2024-06-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -186,3 +186,3 @@
 [dependencies.itertools]
-version = "0.11"
+version = ">= 0.10, < 1.0"
 
@@ -197,3 +197,3 @@
 [dependencies.strum]
-version = "0.25"
+version = "0.26"
 features = ["derive"]
@@ -205,6 +205,2 @@
 
-## enables the [`TermwizBackend`] backend and adds a dependency on the [Termwiz crate].
-[dependencies.termwiz]
-version = "0.20.0"
-optional = true
 
@@ -216,3 +212,3 @@
 [dependencies.unicode-segmentation]
-version = "1.10"
+version = "1.9"
 
